repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/rackspaceauthuser/RackspaceAuthUserTest.groovy

```diff
@@ -194,4 +194,72 @@ class RackspaceAuthUserTest extends ReposeValveTest {
         and: "The result will be passed through"
         messageChain.receivedResponse.code == "200"
     }
+
+    @Unroll("Will parse Forgot Password request for username when sent to appropriate URL (#testName)")
+    def "Will parse a Forgot Password request if the URL matches /v2.0/users/RAX-AUTH/forgot-pwd"() {
+        when: "Request body contains forgot password credentials"
+        def messageChain = deproxy.makeRequest([url: reposeEndpoint + "/v2.0/users/RAX-AUTH/forgot-pwd", requestBody: requestBody, headers: contentType, method: "POST"])
+        def sentRequest = ((MessageChain) messageChain).getHandlings()[0]
```

Comment:
I assume Bill copy/pasta'd this from the previous lines which did this all over.  Why this was done before is... well... the old Groovy code isn't that great.  I will clean some of this mess up.